pack/Microsoft.Azure.Functions.V2.PythonWorker.nuspec

```diff
@@ -18,7 +18,11 @@
     <file src="..\3.7_WINDOWS_X86\**" target="tools\3.7\WINDOWS\X86" />
     <file src="..\3.7_LINUX_X64\**" target="tools\3.7\LINUX\X64" />
     <file src="..\3.7_OSX_X64\**" target="tools\3.7\OSX\X64" />
+    <!-- <file src="..\3.8_WINDOWS_X64\**" target="tools\3.8\WINDOWS\X64" /> -->
```

Comment:
nit: We can just remove these. Is there a reason why you wanted to leave these as comments?

Reply:
Yep, I can remove it, just for the ease of comparison with v3 python worker during development.